### PR TITLE
Add latest version # to all dependencies

### PR DESCRIPTION
**Pull request by simongranger**
Good practice to avoid having "latest" as a version #

### CODE REVIEW

**Comment by HeloiseRD** [comment]
Should we just put "latest" in?

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "keywords": [
     "appium"
   ],
-  "version": "1.0.9",
+  "version": "1.0.11",
```
Comment (simongranger) [inline package.json]: I am skipping a version # because I did another publish as a test

**Comment by HeloiseRD** [comment]
Can you test with 11, see if that works and then bump it to 12? Thanks!

**Comment by pfoster-youitv** [comment]
+1

**Pull request merged**